### ISSUE TITLE
Juniper appliance is broken

### ISSUE THREAD

**Issue by noplay** [GNS3/gns3-registry#82]
https://gns3.com/qa/juniper-vsrx-appliances-template

**Comment by noplay** [comment]
The problem is perhaps inside GNS3 OVA file support

**Comment by ehlers** [comment]
Tested with 1.4.0 on Linux and it indeed doesn't work. Then tested with the current 1.4.1dev2 and it works, so for me it's already fixed. Just tested with local linux server, not with GNS3 VM (I don't use it, I have my local Linux :-).


Found only one problem with the new kvm check:

On Debian (and maybe some other linux variants) the kvm-ok program doesn't exist. Therefore the capabilities don't include kvm. So GNS3 thinks that my linux doesn't support kvm and refuses to import this appliance. By creating a dummy kvm-ok I was able to use the appliance.

I suggest to replace the check for kvm-ok (in modules/qemu/__init__.py) by the check for the existence of /dev/kvm as it's done in modules/qemu/qemu_vm.py .Then it should work on any linux.
